cmd/oras/root/blob/fetch.go

```diff
@@ -45,8 +45,8 @@ func fetchCmd() *cobra.Command {
 	var opts fetchBlobOptions
 	cmd := &cobra.Command{
 		Use:   "fetch [flags] {--output <file> | --descriptor} <name>@<digest>",
-		Short: "Fetch a blob from a remote registry",
-		Long: `Fetch a blob from a remote registry
+		Short: "Fetch a blob from a registry or an OCI layout folder",
```

Comment:
```suggestion
		Short: "Fetch a blob from a registry or an OCI image layout folder",
```

Reply:
We will create a glossary document to explain every terminology. I would suggest using the OCI image layout here by following the [official term](https://github.com/opencontainers/image-spec/blob/main/image-layout.md).